### ISSUE TITLE
[suggestion] save more settings

### ISSUE THREAD

**Issue by axredneck** [guillaumechereau/goxel#172]
e.g.:
- Selected palette
- Light settings
- View settings
and maybe:
- Material
- Some default image settings

Currently only theme is saved, and it's not enough.

**Comment by guillaumechereau** [comment]
I agree.  I think most should be saved in the image itself.  I'll see how to do that.

**Comment by axredneck** [comment]
> I think most should be saved in the image itself

But what about default settings which will be used for new images?

**Comment by YBPN** [comment]
I’d like to add to this. Customizable default settings would be awesome, including for tools.
